Library/Homebrew/cmd/update-report.rb

```diff
@@ -30,6 +30,8 @@ def update_report
         ohai "Homebrew has enabled anonymous aggregate user behaviour analytics"
         puts "Read the analytics documentation (and how to opt-out) here:"
         puts "  https://git.io/brew-analytics"
+        puts "Sleeping for 10 seconds. Press CTRL+C if you'd like to configure analytics"
```

Comment:
You don't have to press Ctrl+C; analytics are disabled the first time you see this warning. You can wait for the update to finish instead.